client/src/components/annotators/ImageAnnotator.vue

```diff
@@ -12,8 +12,23 @@ export default {
       required: true,
     },
   },
-
+  data() {
+    return {
+      loadingVideo: false,
+    };
+  },
   created() {
+    this.playCache = 1; // seconds required to remain playing (should be less that cache seconds)
+    this.cacheSeconds = 3; // seconds to cache
+    this.frontBackRatio = 0.90; // 90% forward frames, 10% backward frames when caching
+    // playCache needs to be less than the adjusted cache seconds to work properly
+    if (this.playCache > this.cacheSeconds * this.frontBackRatio) {
+      const difference = (this.playCache - (this.cacheSeconds * this.frontBackRatio));
+      this.cacheSeconds = Math.ceil(this.cacheSeconds + difference);
+    }
+    if (this.frontBackRatio > 1 || this.frontBackRatio < 0) {
+      this.frontBackRatio = 1.0;
+    }
```

Comment:
You explicitly set it to `0.9`on line 23, then put a conditional here.  I don't understand how the value could have changed in the middle.

Reply:
This wasn't a complete PR thing I wanted the ability to change the cache to see what would work best for our purposes.  That just ensures that I don't  go outside my bounds.  I'm still in the testing phase of what I want to use for the value or if it should be exposed.